ee/modules/030-cloud-provider-zvirt/templates/csi/secret.yaml

```diff
@@ -0,0 +1,18 @@
+{{- define "csi_cloud_config" }}
+  {{- $providerClusterConfiguration := .Values.cloudProviderZvirt.internal.providerClusterConfiguration | required "internal.providerClusterConfiguration is required" -}}
+ovirt_url: {{ $providerClusterConfiguration.provider.server }}
+ovirt_username: {{ $providerClusterConfiguration.provider.username }}
+ovirt_password: {{ $providerClusterConfiguration.provider.password }}
+ovirt_insecure: {{ $providerClusterConfiguration.provider.insecure }}
+clusterid: NO_RDE_{{ .Values.global.discovery.clusterUUID }}
```

Comment:
RDE doesn't belong here, it's vmware thing for collecting events within infrastructure